src/cms/templates/events/event_list_archived.html

```diff
@@ -3,23 +3,51 @@
 {% load static %}
 {% load content_filters %}
 
+{% block css %}
+    <link href="{% static 'tablesort/tablesort.css' %}" rel="stylesheet" type="text/css">
+    {# The following style fixes issues with the arrow denoting the sorting order being wrapped to the next line #}
+    <style>
+        th[role=columnheader]:not(.no-sort) {
+            padding-right: 20px;
+        }
+        th[role=columnheader]:not(.no-sort):after {
+            margin-right: -20px;
+        }
+    </style>
+{% endblock %}
+
 {% block javascript_head %}
     <script src="{% static 'js/confirmation-popups.js' %}"></script>
 {% endblock %}
 
 {% block content %}
     <div class="table-header">
         <div class="flex flex-wrap">
-            <div class="w-1">
+            <div class="w-full">
                 <h2 class="heading font-normal mb-2">{% trans 'Archived events' %}</h2>
             </div>
         </div>
+        <div class="flex flex-wrap">
+            <form class="table-search relative w-1/2">
+                <i data-feather="search" class="absolute"></i>
+                <input type="search" placeholder="{% trans 'Search' %}" class="h-full pl-10 pr-4 rounded shadow">
+            </form>
+            <div class="w-1/2 flex flex-wrap justify-end">
+                <button id="filter-toggle"
+                        class="bg-red-500 hover:bg-red-600 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline mr-3">
+                    {% trans 'Filter events' %}
+                </button>
+            </div>
+        </div>
+    </div>
+    <div id="filter-form-container" class="hidden w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
```

Comment:
```suggestion
    <div id="filter-form-container" class="{% if not filter_form.data %}hidden{% endif %} w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
```
If you add the same suggestion in the non-archived list, you might as well add it here.